nix/sources.nix

```diff
@@ -2,7 +2,7 @@
 # in a flake.lock.
 { system }:
 let
-  lock = builtins.fromJSON (builtins.readFile ../flake.lock);
+  lock = builtins.fromJSON (builtins.readFile ../old_flake.lock);
```

Comment:
old_flake.nix and old_flake.lock will be kept until nix-shell becomes obsolete

Reply:
Needs a comment